docker/ghost-s3/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ghost:3.1 
+
+# inspired by: https://github.com/macchie/docker-ghost-scs/blob/master/Dockerfile
+# https://github.com/colinmeinke/ghost-storage-adapter-s3
+RUN mkdir -p ./content.orig/adapters/storage
```

Comment:
change this from `./` to just using a relative path

Reply:
also have we any way of testing this image?

Reply:
I can try running it locally, setting the NODE_ENV to development and see if it works. Once someone sets up the s3 bucket we can test to make sure that that is working too in dev before deploying this

Reply:
ok. no worries if there's no good story on local testing. i was just curious. local testing is our biggest hole right now throughout our infrastructure.

Reply:
I ran the Docker image locally and it all works as expected.